HyperV/scripts/export-eventlog.ps1

```diff
@@ -1,27 +1,44 @@
-function dumpeventlog(){
+# Loading config
 
-	get-eventlog -list | ForEach-Object {
+. "C:\OpenStack\nova-ci\HyperV\scripts\config.ps1"
+
+function dumpeventlog($path){
+	
+	Get-Eventlog -list | ForEach-Object {
 		$logFileName = $_.LogDisplayName
-		$path = "C:\OpenStack\Logs\Eventlog\"
-		$exportFileName = "eventlog_" + $logFileName + (get-date -f yyyyMMdd) + ".evt"
+		$exportFileName = "\eventlog_" + $logFileName + (get-date -f yyyyMMdd) + ".evt"
```

Comment:
why not set the full pathname here instead of setting here "\filename" and few lines later ($path + $exportfilename)?